html/sites/all/modules/hr/hr_assessments/src/Plugin/resource/RestfulEntityNodeAssessments.php

```diff
@@ -85,14 +85,20 @@ public function publicFields() {
 
     $public_fields['subject'] = array(
       'property' => 'field_asst_subject',
+      'sub_property' => 'value',
+      'process_callbacks' => array('strip_tags', 'trim'),
     );
 
     $public_fields['methodology'] = array(
       'property' => 'field_asst_methodology',
+      'sub_property' => 'value',
+      'process_callbacks' => array('strip_tags', 'trim'),
     );
 
     $public_fields['key_findings'] = array(
       'property' => 'field_asst_key_findings',
+      'sub_property' => 'value',
+      'process_callbacks' => array('strip_tags', 'trim'),
```

Comment:
I *think* that matches the documentation I found, which is a bit thin on _process_callbacks_.